app/assets/scripts/components/upload-form.js

```diff
@@ -125,59 +124,67 @@ var UploadForm = React.createClass({
-            </div>
-          </header>
-          <div className='fold__body'>
-            The CSV parser is able to recognize the following attribute names and characteristics:
-            (An example schema should be included, potentially as a hidden-by-default accordian list)
-          </div>
-
-          <fieldset className='form__fieldset'>
+        <div className='exhibit'>
+          <div className="exhibit__content">
+            {showModal ? <FailureModal errors={errors} /> : ''}
+            {showModal ? <SuccessModal visible={showModal} metadata={this.state.metadata} errors={errors} csvFile={this.csvFile} /> : ''}
+            <div className='inner'>
+              <fieldset className='form__fieldset'>
 
-            <div className='form__group'>
-              <label className='form__label' htmlFor='form-input'>Please select a CSV file</label>
-              <input type='file' className='form__control' id='form-file' accept='text/plain' onChange={this.storeCsv} />
-              <div className='form__input-group'>
-                <span className='form__input-group-button'><button type='submit' className='button button--base button--medium button--example-icon'><label htmlFor='form-file'>Choose File</label></button></span>
-                <input type='text' className='form__control form__control--medium' id='form-input' placeholder='No file chosen' />
-              </div>
-            </div>
+                <div className='form__group form__group--token'>
+                  <label className='form__label' htmlFor='form-input'>Please enter you API token</label>
+                  <p><a className='blue' href='mailto:info@openaq.org'>Don't have a key? Email us to request one.</a></p>
+                  <div className='form__input-group'>
+                    <input type='text' className='form__control form__control--medium' id='form-input' placeholder='Enter Key' />
+                  </div>
+                </div>
 
-            <div className='form__group'>
-              <label className='form__label' htmlFor='form-input'>Please enter you API token</label>
-              <div className='form__input-group'>
-                <input type='text' className='form__control form__control--medium' id='form-input' placeholder='Please enter your uploader API token' />
-              </div>
-            </div>
+                <div className='form__group form__group--upload'>
+                  <label className='form__label' htmlFor='form-input'>Upload Data</label>
+                  <p>We only accept CSV files at this time.</p>
+                  <input type='file' className='form__control--upload' id='form-file' accept='text/plain' onChange={this.getFile} />
+                  <div className='form__input-group'>
+                    <span className='form__input-group-button'><button type='submit' className='button button--base button--medium button--arrow-up-icon'><label htmlFor='form-file'>Upload</label></button></span>
```

Comment:
@danielfdsilva I'm seeing examples of using jQuery to simulate a click on file input, but I can't find any native code solutions. Any idea?
Actually, the two solutions I've found are the label method I'm using, and then a trick of absolutely positioning an opacity 0 input button over a fake button, which seems like it would be even worse.